examples/apollo-watch-fragments/src/Todo.tsx

```diff
@@ -1,24 +1,27 @@
 import React, { useCallback } from "react";
 import {
-  useFragment,
+  useRefetachbleFragment,
   shallowCompareFragmentReferences,
 } from "@graphitation/apollo-react-relay-duct-tape";
 import { graphql } from "@graphitation/graphql-js-tag";
 
 import useChangeTodoStatusMutation from "./useChangeTodoStatusMutation";
 
 import { Todo_todoFragment$key } from "./__generated__/Todo_todoFragment.graphql";
+import { TodoRefetchQuery } from "./__generated__/TodoRefetchQuery.graphql";
 
 export const Todo_todoFragment = graphql`
-  fragment Todo_todoFragment on Todo {
+  fragment Todo_todoFragment on Todo
+  @refetchable(queryName: "TodoRefetchQuery") {
     id
     description
     isCompleted
+    someOtherField @include(if: $includeSomeOtherField)
```

Comment:
This is truly a horribly uninspired example. Need to come up with a better one that makes sense in the context of this todo app, but for now it at least demonstrates its working.